symplyphysics/laws/waves/photon_momentum_is_proportional_to_propagation_vec.py

```diff
@@ -0,0 +1,36 @@
+from sympy import (Eq, solve, pi)
+from sympy.physics.units import planck as planck_constant
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## Photon is the elementary part of any electromagnetical radiation which has no mass and always moves with speed of light.
+## In despite of no mass, it carries some momentum. The amount of this momentum depends only on the propagation vector of the photon.
+## Law is: p = (h/2pi) * k, where
+## p is momentum of photon,
+## h is Planck constant,
+## k is module of the propagation vector of photon.
+
+photon_momentum = Symbol("photon_momentum", units.momentum)
+module_propagation_vector = Symbol("module_propagation_vector", 1/units.length)
+
+law = Eq(photon_momentum, (planck_constant/(2*pi)) * module_propagation_vector)
+
+
+def print_law() -> str:
+    return print_expression(law)
+
+
+@validate_input(photon_propagation_=module_propagation_vector)
+@validate_output(photon_momentum)
+def calculate_momentum(photon_propagation_: Quantity) -> Quantity:
```

Comment:
Actually I found a proper name for this stuff: https://en.m.wikipedia.org/wiki/Wavenumber
Let's use it instead of propagation_vector, as soon as direction of the wave is unrelated for this law.